### PR TITLE
[4.1] Update RemoveUserFromGroupCommand.php

### PR DESCRIPTION
**Pull request by brianteeman**
english language correction - review

### before
image

### after
image

### CODE REVIEW

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on dbdd731cd6d4e9744bd82e533e5709e13e9c6c5d<br /><br />Tested successfully in Joomla 4.1.3-dev of 24 April in Wampserver 3.2.8 using PHP 8.0.15<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37648">issues.joomla.org/tracker/joomla-cms/37648</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on fc6d246d47c557ee6fb1702ec2d3ba3d97cd2484<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37648">issues.joomla.org/tracker/joomla-cms/37648</a>.</sub>

File: libraries/src/Console/RemoveUserFromGroupCommand.php [hunk 1]
```diff
@@ -141,7 +141,9 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
 
 			if (\count(Access::getGroupsByUser($user->id, false)) < 2)
 			{
-				$this->ioStyle->error("Can't remove '" . $user->username . "' from group '" . $result . "'! Every user needs at least one group");
+				$this->ioStyle->error("Can't remove '" . $user->username . "' from group '" . $result . "
+					'! Every user needs to be a member of at least one group"
```
Comment (richard67) [inline libraries/src/Console/RemoveUserFromGroupCommand.php]:
```suggestion
				$this->ioStyle->error("Can't remove '" . $user->username . "' from group '" . $result
					. "'! Every user needs to be a member of at least one group"
```
I would not do the line break within the sting.

Comment (brianteeman) [reply libraries/src/Console/RemoveUserFromGroupCommand.php]: I didnt want to either but it gave a line lenth error and this change is consistent with line 134

Comment (richard67) [reply libraries/src/Console/RemoveUserFromGroupCommand.php]: @brianteeman In line 134 the new line is not done within the double quotes and so not within the string. It is exactly there like I suggest here, the line ends with a sting end (double-quote in this case) and the next line starts with the concatenation operator. See https://github.com/joomla/joomla-cms/blob/fc6d246d47c557ee6fb1702ec2d3ba3d97cd2484/libraries/src/Console/RemoveUserFromGroupCommand.php#L134 :

```
$this->ioStyle->error("Can't remove user '" . $user->username . "' from group '" . $result . "'! "
	. $result . " needs at least one active user!"
);
```

Comment (richard67) [reply libraries/src/Console/RemoveUserFromGroupCommand.php]: Your code currently does the line change between the double quotes and so in the middle of a string and that's not the case in line 134.

**Comment by richard67** [comment]
I've restored the previous results from human test in the issue tracker because the commits which invalidated them were just code style changes.

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37648">issues.joomla.org/tracker/joomla-cms/37648</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
thanks

**Pull request merged**